### PR TITLE
feat(scheduler): scale the extender output in generic scheduler

### PR DESCRIPTION
**Pull request by draveness**
/kind cleanup
/priority important-soon
/assign @ahg-g 

**What this PR does / why we need it**:

scale the extender output in generic scheduler

**Which issue(s) this PR fixes**:

ref https://github.com/kubernetes/kubernetes/issues/81281

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/scheduler/apis/extender/v1/types.go [hunk 1]
```diff
@@ -23,10 +23,10 @@ import (
 
 const (
 	// MinExtenderPriority defines the min priority value for extender.
-	MinExtenderPriority int = 0
+	MinExtenderPriority int64 = 0
```
Comment (draveness) [inline pkg/scheduler/apis/extender/v1/types.go]: keep consistent with the type of `MaxNodeScore`

File: pkg/scheduler/core/generic_scheduler.go [hunk 2]
```diff
@@ -840,7 +840,7 @@ func PrioritizeNodes(
 		// wait for all go routines to finish
 		wg.Wait()
 		for i := range result {
-			result[i].Score += combinedScores[result[i].Name]
+			result[i].Score += combinedScores[result[i].Name] * (framework.MaxNodeScore / extenderv1.MaxExtenderPriority)
```
Comment (ahg-g) [inline pkg/scheduler/core/generic_scheduler.go]: Please add a comment saying that: "MaxExtenderPriority may diverge from the max priority used in the scheduler and defined by MaxNodeScore, therefore we need to scale the score returned by extenders to the score range used by the scheduler".

**Comment by draveness** [comment]
/retest

Regards,
Draven

File: pkg/scheduler/core/generic_scheduler.go (hunk 2 above)
Comment (draveness) [reply pkg/scheduler/core/generic_scheduler.go]: done

**Comment by ahg-g** [comment]
/lgtm
/approve

Thanks!

**Comment by draveness** [comment]
/retest

Regards,
Draven

**Pull request merged**